app/controller/sdl/RCModulesController.js

```diff
@@ -0,0 +1,946 @@
+                        }
+                        if(data.params.moduleData.audioControlData.source === 'MOBILE_APP') {
+                          FFW.RC.OnIVDNotificationWasSent = true;
+                        }
+                        dataToReturn.audioControlData = audioControlData;
+                    }
+                }
+                break;
+            }
+            case 'HMI_SETTINGS':
+            {
+                if(data.params.moduleData.hmiSettingsControlData){
+                    var hmiSettingsControlData = this.hmiSettingsModels[moduleId].setHmiSettingsData(
+                      data.params.moduleData.hmiSettingsControlData);
+                    if (Object.keys(data.params.moduleData.hmiSettingsControlData).length > 0) {
+                    FFW.RC.onInteriorVehicleDataNotification({moduleType:'HMI_SETTINGS', moduleId: moduleUUId,
+                                                                hmiSettingsControlData: hmiSettingsControlData});
+                    }
+                    dataToReturn.hmiSettingsControlData = hmiSettingsControlData;
+                }
+                break;
+            }
+            case 'LIGHT':
+            {
+                if(data.params.moduleData.lightControlData){
+                    var lightControlData = this.lightModels[moduleId].setLightControlData(
+                      data.params.moduleData.lightControlData);
+
+                    if (Object.keys(lightControlData).length > 0) {
+                    FFW.RC.onInteriorVehicleDataNotification({moduleType:'LIGHT', moduleId: moduleUUId,
+                                                                lightControlData: data.params.moduleData.lightControlData});
+                    }
+                    dataToReturn.lightControlData = lightControlData;
+                }
+                break;
+            }
+            case 'SEAT':
+            {
+                if(data.params.moduleData.seatControlData){
+                    var seatControlData = this.seatModels[moduleId].setSeatControlData(
+                    data.params.moduleData.seatControlData);
+                    if (Object.keys(data.params.moduleData.seatControlData).length > 0) {
+                    FFW.RC.onInteriorVehicleDataNotification({moduleType:'SEAT', moduleId: moduleUUId,
+                                                                seatControlData: seatControlData});
+                    }
+                    dataToReturn.seatControlData = seatControlData;
+                }
+            }
+        }
+        return dataToReturn;
+    },
+
+    /**
+     * @function getInteriorVehicleData
+     * @param {Object} data
+     * @description get data by moduleId and moduleType
+     */
+    getInteriorVehicleData: function(data) {
+        var moduleUUId = data.params.moduleId;
+        var moduleType = data.params.moduleType;
+
+        var moduleId = null;
```

Comment:
The second identifier used for the `moduleUUIDMapping` is a location grid coordinate as a string `L<level>R<row>C<col>` so `moduleId` should be renamed to 
```suggestion
        var location_name = null;
```

Reply:
@ShobhitAd accepted. Fixed in https://github.com/smartdevicelink/sdl_hmi/commit/3356c1d8aa1ec7b881f8a7e25bb4cde2a50033af